tensorflow/tensorboard/functionaltests/basic_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Basic TensorBoard functional tests using WebDriver."""
+
+import os
```

Comment:
Turns out that I just needed to add srcs_version=PY2AND3 on all the python targets in io_bazel_rules_web and this target.